packages/plugins/GoPlusSecurity/src/SNSAdaptor/index.tsx

```diff
@@ -1,29 +1,41 @@
 import type { Plugin } from '@masknet/plugin-infra'
+import { PluginI18NFieldRender } from '@masknet/plugin-infra/content-script'
 import { base } from '../base'
 import { CheckSecurityDialog } from './CheckSecurityDialog'
 import { useState } from 'react'
 import { ApplicationEntry } from '@masknet/shared'
+import { SecurityCheckerIcon } from '@masknet/icons'
 
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     init(signal) {},
     ApplicationEntries: [
-        {
-            RenderEntryComponent() {
-                const [open, setOpen] = useState(false)
-                return (
-                    <>
-                        <ApplicationEntry
-                            title="Check Security"
-                            icon={new URL('../assets/security-icon.png', import.meta.url).toString()}
-                            onClick={() => setOpen(true)}
-                        />
-                        {open && <CheckSecurityDialog open={open} onClose={() => setOpen(false)} />}
-                    </>
-                )
-            },
-            defaultSortingPriority: 12,
-        },
+        (() => {
+            const icon = <SecurityCheckerIcon />
+            const name = { i18nKey: '__plugin_name', fallback: 'Check Security' }
+
+            return {
+                ApplicationEntryID: base.ID,
+                RenderEntryComponent({ disabled }) {
+                    const [open, setOpen] = useState(false)
+                    return (
+                        <>
+                            <ApplicationEntry
+                                title={<PluginI18NFieldRender field={name} pluginID={base.ID} />}
```

Comment:
You could just use `useI18N` exported from '../locales/' instead of using PluginI18NFieldRender.

Reply:
Strange, useI18N() does not work.